test/test_entry_filter.rb

```diff
@@ -82,6 +82,9 @@ class TestEntryFilter < JekyllUnitTest
 
     # rubocop:disable Performance/FixedSize
     should "include only safe symlinks in safe mode" do
+      # no support for symlinks on Windows
+      skip_if_windows
```

Comment:
Same here and everywhere you use this method – make that comment the message instead of a comment so when I run the tests, this information is surfaced.